packages/fluentui/e2e/tests/chatMessageActionMenu.spec.ts

```diff
@@ -12,7 +12,7 @@ describe('Chat message with action menu rendered outside', () => {
       selectors.chatMessageClassName
     }`;
 
-  const getActionMenuAt = index => `${selectors.menuClassName}:nth-child(${index + 1})`;
+  const getActionMenuAt = index => `#action-menu${index}`;
```

Comment:
Before this test was relying on DOM structure:

```html
<div class="ui-provider">
  <div class="ui-menu"></div>
  <div class="ui-menu"></div>
</div>
```

- I added explicit IDs instead
- It's the reason why changes are marked as breaking